[wayland] Unable to build from source

```
config.status: linking protocol/wayland.dtd to src/wayland.dtd.embed
config.status: executing depfiles commands
config.status: executing libtool commands
  CC       src/wayland_scanner-scanner.o
  CPPAS    src/wayland_scanner-dtddata.o
  CC       src/libwayland_util_la-wayland-util.lo
In file included from /usr/local/include/libxml2/libxml/parser.h:810,
                 from src/scanner.c:42:
/usr/local/include/libxml2/libxml/encoding.h:28:10: fatal error: iconv.h: No such file or directory
   28 | #include <iconv.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [Makefile:1963: src/wayland_scanner-scanner.o] Error 1
make: *** Waiting for unfinished jobs....
wayland failed to install: `make V=0 -j4` exited with 2
```

A reinstall of glibc should fix the problem, I think...

On Sun, Jan 24, 2021, 9:25 AM Ed Reel <notifications@github.com> wrote:

> config.status: linking protocol/wayland.dtd to src/wayland.dtd.embed
> config.status: executing depfiles commands
> config.status: executing libtool commands
>   CC       src/wayland_scanner-scanner.o
>   CPPAS    src/wayland_scanner-dtddata.o
>   CC       src/libwayland_util_la-wayland-util.lo
> In file included from /usr/local/include/libxml2/libxml/parser.h:810,
>                  from src/scanner.c:42:
> /usr/local/include/libxml2/libxml/encoding.h:28:10: fatal error: iconv.h: No such file or directory
>    28 | #include <iconv.h>
>       |          ^~~~~~~~~
> compilation terminated.
> make: *** [Makefile:1963: src/wayland_scanner-scanner.o] Error 1
> make: *** Waiting for unfinished jobs....
> wayland failed to install: `make V=0 -j4` exited with 2
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/skycocker/chromebrew/issues/4985>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAILX7NC6WLD37U5CJISNP3S3QUVXANCNFSM4WQSJKQA>
> .
>

--- a/packages/wayland.rb
+++ b/packages/wayland.rb
@@ -8,6 +8,19 @@ class Wayland < Package
   source_url 'https://wayland.freedesktop.org/releases/wayland-1.18.0.tar.xz'
   source_sha256 '4675a79f091020817a98fd0484e7208c8762242266967f55a67776936c2e294d'
 
+  binary_url ({
+     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/wayland-1.18.0-2-chromeos-armv7l.tar.xz',
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/wayland-1.18.0-2-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/wayland-1.18.0-2-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/wayland-1.18.0-2-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+     aarch64: 'f5686b065f2395724aadbd5c27d953aa9a93035f35fd71b315fcddf96c89507a',
+      armv7l: 'f5686b065f2395724aadbd5c27d953aa9a93035f35fd71b315fcddf96c89507a',
+        i686: '1abcde23427ad7fffc9773cca1695790ec08e7a0ee2b98c4601f395a808acac4',
+      x86_64: '41abcb4d0bf10e5b09b8cecdcfe8685379d91b07e541ed54654477c8fbb807fb',
+  })
+
   depends_on 'expat'
   depends_on 'libpng'
   depends_on 'libffi'

--- a/packages/wayland_protocols.rb
+++ b/packages/wayland_protocols.rb
@@ -8,6 +8,19 @@ class Wayland_protocols < Package
   source_url 'https://wayland.freedesktop.org/releases/wayland-protocols-1.20.tar.xz'
   source_sha256 '9782b7a1a863d82d7c92478497d13c758f52e7da4f197aa16443f73de77e4de7'
 
+  binary_url ({
+     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/wayland_protocols-1.20-1-chromeos-armv7l.tar.xz',
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/wayland_protocols-1.20-1-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/wayland_protocols-1.20-1-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/wayland_protocols-1.20-1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+     aarch64: '1decd76dce195360e3b2879b7a1ee63efefc64b437faa98e9ff5c833758730d2',
+      armv7l: '1decd76dce195360e3b2879b7a1ee63efefc64b437faa98e9ff5c833758730d2',
+        i686: '6ff5c74feef6541c342f7393fb2df70e4ec121f4b94d55f254bfa28c6e3e3dc0',
+      x86_64: 'a88f0f3c10c6108c6b9b584a1c74998d8a49ea70cb022f6350d3df129f120a72',
+  })
+
   depends_on 'wayland'
 
   def self.build

